test/integration/tsdx-build-withBabel.test.js

```diff
@@ -0,0 +1,58 @@
+const shell = require('shelljs');
+
+const util = require('../util');
+
+shell.config.silent = false;
+
+const testDir = 'integration';
+const fixtureName = 'build-withBabel';
+const stageName = `stage-integration-${fixtureName}`;
+
+describe('integration :: tsdx build :: .babelrc.js', () => {
+  beforeAll(() => {
+    util.teardownStage(stageName);
+    util.setupStageWithFixture(testDir, stageName, fixtureName);
+  });
+
+  it('should convert styled-components template tags', () => {
+    let output = shell.exec('node ../dist/index.js build');
+    expect(output.code).toBe(0);
+
+    // from styled.h1` to styled.h1(
+    output = shell.grep(/styled.h1\(/, ['dist/build-withbabel.*.js']);
+    expect(output.code).toBe(0);
+  });
```

Comment:
well this isn't exactly a babel integration test, only the `babel-plugin-styled-components` one is, but I guess it's also a pre-req... maybe I should merge this with the test below?

Reply:
Or, I guess template tags are babel since it's a newer syntax. Will leave it as is, nbd either way.